web/src/components/Features.js

```diff
@@ -88,7 +89,7 @@ const Features = () => {
           title="Full-stack Auth"
           url="/blog/2023/04/12/auth-ui"
           description={`
-            Add login with social providers or email in a few lines of code with powerful UI helpers. No third party vendor lock-in.
+            Add email and social login in a few lines of code. Wasp manages everything from sessions to UI components. No boilerplate, no lock-in.
```

Comment:
Maybe let's make it clearer it handles full stack? sessions to UI components -> sessions kind of indicate it is handling backend stuff, but I have to admit I myself woudln't be sure what that means all together. I would say "Wasp handles everything from UI components to sessions in the database, full stack.", something like that.